src/NuGet.Clients/PackageManagement.UI/Models/SearchResultPackageMetadata.cs

```diff
@@ -104,4 +217,13 @@ public override string ToString()
             return Id;
         }
     }
-}
+
+    internal class BackgroundLoaderResult
+    {
+        public PackageStatus Status { get; set; }
+
+        public NuGetVersion LatestVersion { get; set; }
+
+        public NuGetVersion InstalledVersion { get; set; }
+    }
+}
```

Comment:
Need to deal with cases of failure to retrieve data from the server. (Probably need a status in package status as well)